vllm/spec_decode/selector.py

```diff
@@ -0,0 +1,53 @@
+from vllm.platforms import current_platform
+
+if current_platform.is_neuron():
+    from vllm.worker.neuron_model_runner import (  # noqa: F401
+        ModelInputForNeuron as ModelInputCls)
+    from vllm.worker.neuron_model_runner import (  # noqa: F401
+        NeuronModelRunner as ModelRunnerCls)
+    from vllm.worker.neuron_worker import (  # noqa: F401
```

Comment:
oh I actually plan to add some arguments like `--worker-cls auto` and let every platform select there own worker class. we should do that.

Reply:
@youkaichao, is something I can refer to? Or is this file works, currently, I put it under spec_decode folder, it also makes sense to put under worker folder.